CLAUDE.md

```
@@ -0,0 +1,113 @@
+# CLAUDE.md
+
+이 파일은 Claude Code(claude.ai/code)가 이 리포지토리의 코드를 작업할 때 참고할 가이드를 제공합니다.
+
+## Project Overview
+
+RoomLog는 iOS SwiftUI 앱입니다 (2026 동국대 정보통신공학과 캡스톤 디자인).
+Xcode를 통해 빌드 및 실행하며, `RoomLog/RoomLog.xcodeproj` 파일을 열어 시작합니다.
+
+## 폴더 구조
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)